tests/drivers/spi/spi_loopback/boards/ucans32k1sic.conf

```diff
@@ -0,0 +1,4 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SPI_ASYNC=n
```

Comment:
How come this is needed? Perhaps mention it in the commit log.

Reply:
a mistake, that overlay is not necessary